Gemfile

```diff
@@ -260,6 +260,12 @@ group :web_socket, :manageiq_default do
   gem "websocket-driver",               "~>0.6.3",           :require => false
 end
 
+if ENV["APPLIANCE"]
+  group :appliance do
+    gem "manageiq-appliance_console",     "~>7.0", ">=7.0.3",  :require => false
```

Comment:
```suggestion
    gem "manageiq-appliance_console",     "~>7.0", ">=7.0.4",  :require => false
```

Reply:
I guess I do wonder if we need the `ENV["APPLIANCE"]`
Something nice about not having conditionals in the `Gemfile`.

Then again, having that conditional is nice because with it, we wouldn't add a gem to `CI` / developer's `gemfile.lock`.